### ISSUE TITLE
See also... (ChatGeoPT and "ChatGPT is now a GIS Tool")

### ISSUE THREAD

**Issue by sfkeller** [dw-innovation/kid2-spot#1]
I am happy to support this project, which I hope will remain open-source and privacy-preserving. That said, it's always good to know what you can do with ChatGPT the "original" (which is not privacy-preserving ).

See also:
* ChatGeoPT - a ChatGPT app (a repo "only"), open source, (March 2023): https://medium.com/earthrisemedia/chatgeopt-exploring-the-future-of-talking-to-our-maps-b1f82903bb05
* ChatGPT Plus, the "original" (August 2023), e.g. "ChatGPT is now a GIS Tool": Video www.youtube.com/watch?v=dgzWLBYswh0

**Comment by laem** [comment]
Hi, thanks for your work ! I'm wondering how your results are different from training a LLM to build overpassal queries. I've been trying it with chatgpt and Claude, both can easily answer to questions like "find bars in Rennes with a terrace".

**Comment by sfkeller** [comment]
HI @laem: The work I mentoned under "See also" is not from me. I'm working on a more specific project called "AI-assisted Map Styler" to thoose colors for styling any geospatial data. See QGIS Plugin AIAMAS https://github.com/qgis/QGIS/wiki/QOD-December-2023 .

**Comment by laem** [comment]
Yes, sorry it was unclear, I was reacting to the idea of your first link.

**Comment by sfkeller** [comment]
> Yes, sorry it was unclear, I was reacting to the idea of your first link.

That's something we could ask @bengmstrong .

**Comment by dendelyne** [comment]
Apologies for the late response.
Thank you very much for your input! Our team also made an overview with some additional tools such as GeoGuessr GPT, GeoSpy and EarthKit. You can find a brief article discussing their usability in comparison with Spot here: https://innovation.dw.com/articles/geolocation-verification-tools-comparison :)

**Comment by chrisdebian** [comment]
Hi, all.

Excellent project, I read about on an OSM post. Just wondered if this issue still needs to be open? 

Thanks,

Chris

**Comment by sfkeller** [comment]
You can close the issue if you want.

P.S. A question on this occasion: What am I doing incorrectly or expecting wrong when I zoom in on Lake Zurich at https://www.findthatspot.io/ (somewhere like this https://osm.org/go/0C0PHTD#) and the query “Show all restaurants near a lake (shore)” returns “No results in search area”?
